core/templates/components/certificate-assessment-offering-helper/certificate-offering-confirmation-modal.component.html

```
@@ -0,0 +1,33 @@
+    <p class="mb-2"><strong>Save as Not Ready</strong></p>
+    <p class="m-0 mb-2">The certificate will be saved but will not be visible to learners.</p>
+    <p class="m-0">You can make it available later once everything is complete.</p>
+  </div>
+
+  <p class="mb-0">
+    Note: Certificates can only be made available when the question pool meets the required criteria.
+  </p>
+</div>
+
+<div class="modal-footer">
+  <button class="btn btn-secondary" (click)="cancel()">Cancel</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)